script/script.js

```diff
@@ -12,4 +12,202 @@ for (let i = 0; i < links.length; i += 1) {
+              standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s 
+              with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's 
+              standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax 
+              map lapora verita.
+              `,
+    LiveLink: '',
+    SourceLink: '',
+    image: './img/Pic.png',
+    privateClasses: {
+      seeProjectBtn: 'see-projec-btn-1',
+      modalContainer: 'modal-container1',
+      modalCloseBtn: 'modal-close-btn1',
+    },
+  },
+];
+document.addEventListener('DOMContentLoaded', () => {
+  const projectsContainer = document.querySelector('.posts');
+  let projectContent;
+  let projectLanguages;
+  let projectCardContainer;
+  let modalContent;
+  let modalContainer;
+  projects.forEach((project) => {
+    projectLanguages = project.technologies.map(
+      (lang) => `<li>${lang}</li>`,
+    );
+    projectContent = `
+                <h2>${project.title}</h2>
+                <p>${project.description}</p>
+                <ul class="programming-languages-1">
+                  ${projectLanguages.join('')}
+                </ul>
+                <a 
+                class="button-2 ${project.privateClasses.seeProjectBtn}" target="blank">See Project
+                </a>
+              `;
+
+    projectCardContainer = document.createElement('div');
+    projectCardContainer.innerHTML = projectContent;
+    projectCardContainer.setAttribute('class', 'work-pic-2', 'work-lists', 'posts');
+    projectsContainer.appendChild(projectCardContainer);
+
+    modalContent = `
+          <div class="modal-content">
+            <div class="modal-header">
+              <h2>${project.title}</h2>
+              <button class="modal-close-btn ${project.privateClasses.modalCloseBtn}">&times;</button>
+            </div>
+            <ul class="modal-langs">
+              ${projectLanguages.join('')}
+            </ul>
+            <div class="modal-details">
+              <div> 
+                <img  class="modal-image" src=${project.image} alt="Project Image">
+              </div>
+                  <div class="modal-description">
+                    <div>${project.details}</div>
+                    <div class="modal-buttons">
+                      <form action="https://ney2.github.io/Portfolio/" method="get" target="_blank">
+                        <button type="submit"><img src="./img/Enable-1.png" alt="Deployed page"></button>
+                          </form>
+                          <form action="https://github.com/Ney2/Portfolio" method="get" target="_blank">
+                          <button type="submit"><img src="./img/Enable-2.png" alt="Source page"></button>
+                          </form>
```

Comment:
- [ ] I noticed that your entire modal buttons are images. This is not good for responsiveness. Instead of that, simply style the buttons to look exactly like the Figma design using CSS. The only images you should use are the Github and live icons 🙏